crates/ra_project_model/src/lib.rs

```diff
@@ -403,7 +423,7 @@ pub fn get_rustc_cfg_options() -> CfgOptions {
         }
     }
 
-    match (|| -> Result<_> {
+    match (|| -> ::std::result::Result<String, Box<dyn Error + Send + Sync + 'static>> {
```

Comment:
This probably wants to use `anyhow::Error` rather than `Box<dyn Error>` as well.

Reply:
@matklad , maybe `anyhow::Result`, anyhow?

Reply:
done